dev-python/pytest-flask/pytest-flask-1.3.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYPI_NO_NORMALIZE=1
+PYTHON_COMPAT=( python3_{11..13} pypy3 )
+inherit distutils-r1 pypi
+
+DESCRIPTION="A set of pytest fixtures to test Flask applications"
+HOMEPAGE="
+	http://pytest-flask.readthedocs.org
+	https://github.com/pytest-dev/pytest-flask
+	https://pypi.org/project/pytest-flask/
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-python/flask[${PYTHON_USEDEP}]
+	dev-python/werkzeug[${PYTHON_USEDEP}]
+"
+
+DEPEND="${RDEPEND}"
```

Comment:
Do you actually need them to build the wheel?

Reply:
As far as I understand, DEPEND or BDEPEND were meant to be used to declare test dependencies. In this case as the tests also need the same libs than at runtime, it made sense to have DEPEND=$RDEPEND, but they don't seem to be strictly needed for building the package.

Reply:
That's `BDEPEND="test? ( ${RDEPEND} )"`, and `distutils_enable_tests` does it for you.